docs/internal/input-validations-framework-readme.md

```diff
@@ -0,0 +1,92 @@
+# How to use the framework
+
+The constraints framework implementation is located in the utils/constraints-utils.ts module
+
+```typescript
+import * from 'utils/constraints-utils';
+```
+
+## What can you use with the framework
+
+You will find in the constraints framework the generic calls for the following classes if you properly import them from the constraints framework.
```

Comment:
There are no generic calls in the framework. A call is an invocation of a specific function by the client (e.g. any code outside of the framework). 

You can state: the constraints framework provides a set of generic classes and interfaces as well as the invocation framework to validate constraints on arbitrary objects.